includes/framework/QDbBackedSessionHandler.class.php

```diff
@@ -123,6 +123,15 @@ public static function SessionRead($id) {
 			if (!$result_row) // either the data was empty or the row was not found
 				return '';
 			$strData = $result_row['data'];
+
+			if(strstr($objDatabase->Adapter, 'PostgreSql')) {
```

Comment:
@vaibhav-kaushal please, create a new issue with a proper problem explanation and assign it to the 3.1 milestone. After that we would be able to merge this PR with a reference to it from the new issue created.